terraform/aptos-node/gcp/README.md

```diff
@@ -0,0 +1,180 @@
+# Using Terraform
+
+## Run on GCP
+This guide assumes you already have GCP account setup, and have created a new project for deploying Aptos node.
+
+Install pre-requisites if needed:
+
+   * Terraform 1.1.7: https://www.terraform.io/downloads.html
+   * Kubernetes cli: https://kubernetes.io/docs/tasks/tools/
+   * Google Cloud cli: https://cloud.google.com/sdk/docs/install-sdk
+
+1. Create a working directory for your configuration.
+
+    * Choose a workspace name e.g. `testnet`. Note: this defines terraform workspace name, which in turn is used to form resource names.
```

Comment:
workspace name e.g. → workspace name, e.g.,
terraform → Terraform (all occurrences in this PR)